index.bs

```diff
@@ -582,49 +583,45 @@ To <dfn>run the update intersection observations steps</dfn> for an
 
 1. Let |unit| be the <a>unit of related similar-origin browsing contexts</a> for
 	|loop|.
-2. For each |observer| in |unit|'s <a>IntersectionObservers</a> list:
+2. Let |observer list| be a list of all {{IntersectionObserver}}s
+	whose {{IntersectionObserver/root}} is in the DOM tree of a document in |unit|.
+3. For each |observer| in |observer list|:
 	1. Let |rootBounds| be |observer|'s <a>root intersection rectangle</a>.
 	2. For each |target| in |observer|'s internal {{[[ObservationTargets]]}} slot:
-		1. Let |targetRect| be a {{DOMRectReadOnly}}
+		1. If |target| is not a descendant of the <a>intersection root</a>
```

Comment:
This should also only apply if the <a>intersection root</a> is not the <a>implicit root</a>.